stripe/src/main/java/com/stripe/android/auth/PaymentBrowserAuthContract.kt

```diff
@@ -33,14 +34,23 @@ internal class PaymentBrowserAuthContract(
             defaultReturnUrl
         )
 
+        val statusBarColor = when (context) {
+            is Activity -> context.window?.statusBarColor
+            else -> null
+        }
+
+        val extras = input
+            .copy(statusBarColor = statusBarColor)
```

Comment:
can we avoid sending it if it's `null`?

Reply:
`PaymentBrowserAuthContract.Args#statusBarColor` already defaults to null, so it's a no-op if `statusBarColor == null`.